sql/core/src/main/scala/org/apache/spark/sql/catalog/Catalog.scala

```diff
@@ -33,36 +33,37 @@ import org.apache.spark.storage.StorageLevel
 abstract class Catalog {
 
   /**
-   * Returns the current default database in this session.
+   * Returns the current database/namespace in this session.
```

Comment:
If you don't mind, could you avoid to use `/` here? You can use `or` literally. Otherwise, `/` could be read as another multi-layer. :)

Reply:
yeah it becomes tricky to document as
1. namespace could be multiple layers, e.g. ns1.ns2.ns3.ns4
2. I guess technically usually people may think database is a single name (and `.` could be treated as a part of the name than a layer separator).

but keep the `database` will maintain backward understanding as people have get used to it.

I was thinking a few options
1. database or namespace
2. namespace (database)
3. database (namespace).

I am not sure if there are more clear way to document.

Reply:
Since we are not going to rename `database` in the APIs, I'd prefer `database (namespace)`